internal/function_date_test.go

```diff
@@ -0,0 +1,18 @@
+package internal
+
+import (
+	"testing"
+	"time"
+)
+
+func TestDateTrunc(t *testing.T) {
```

Comment:
Please add a query based test case to query_test.go to test the `DATE_TRUNC/DAY` function.
If there is the test case, we don't need this one.

Reply:
👍

Reply:
done @goccy, thanks!